### PR TITLE
Psql

### PR DESCRIPTION
**Pull request by MaxHalleran**
Adds psql support

### CODE REVIEW

**Review by Nolan-E — APPROVED** [review]
I'll approve this, but gonna tag @richardwan for a quick set of eyes too.

File: MyFirstBlog/Migrations/DataContextModelSnapshot.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿// <auto-generated />
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using MyFirstBlog.Helpers;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace MyFirstBlog.Migrations
```
Comment (richardwan) [inline MyFirstBlog/Migrations/DataContextModelSnapshot.cs:10]: what is your plan to run migrations in the prod environment and the CI environment? i was never able to figure that part out

Comment (MaxHalleran) [reply MyFirstBlog/Migrations/DataContextModelSnapshot.cs:10]: Good question!

[Here is documentation on EF Core](https://learn.microsoft.com/en-us/ef/core/cli/dotnet)

[And Here is documentation on migrations in dotnet](https://learn.microsoft.com/en-us/ef/core/managing-schemas/migrations/?tabs=dotnet-core-cli)

I will get back to you once I have figured this out.

File: MyFirstBlog/Helpers/ConnectionHelper.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
+namespace MyFirstBlog.Helpers;
```
Comment (MaxHalleran) [inline MyFirstBlog/Helpers/ConnectionHelper.cs:1]: Added this class to help database connections. Now, the server can be launched on heroku, railway, and potentially cloud66.

File: MyFirstBlog/Program.cs [hunk 3]
```diff
@@ -1,36 +1,46 @@
...
-builder.Services.AddSwaggerGen();
+services.AddEndpointsApiExplorer();
+services.AddSwaggerGen();
+
+services.AddScoped<IPostService, PostService>();
 
 var app = builder.Build();
 
+var scope = app.Services.CreateScope();
+await DataHelper.ManageDataAsync(scope.ServiceProvider);
```
Comment (MaxHalleran) [inline MyFirstBlog/Program.cs:32]: Runs the migrations on program startup. Should this only run in production?

**Review by richardwan — APPROVED** [review]

**Pull request merged**